packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +66,42 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS ? `padding: ${paddingCSS}` : nothing}>
```

Comment:
do you think we can return `padding: ${paddingCSS}` from `generatePaddingCSS()`; directly? so we control the whole generation from one place.  

```suggestion
                style=${paddingCSS ? paddingCSS  : nothing}>
```